chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java

```diff
@@ -37,31 +38,31 @@
 
     @Override
     public void setEnvironment(final Environment environment) {
-        LOGGER.info("setEnvironment");
         this.env = environment;
     }
 
     @Bean
     public Docket customDocket() {
-        LOGGER.info("get Docket");
         return new Docket(DocumentationType.SWAGGER_2).apiInfo(apiInfo()).pathProvider(pathProvider()).select()
                 .paths(this.paths()).build();
     }
 
     private ApiInfo apiInfo() {
-        LOGGER.info("get ApiInfo");
+        Contact contact = new Contact("CHPL Development Team. "
+                + "Please submit any questions using the Health IT Feedback Form and "
+                + "select the \"Certified Health IT Products List (CHPL)\" category.",
+                "https://www.healthit.gov/form/healthit-feedback-form",
+                "");
         return new ApiInfo("CHPL", "Certified Health IT Product Listing", "18.0.0", "http://terms/of/service.url",
-                "CHPL@ainq.com", "License Text", "https://github.com/chpladmin/chpl-api/blob/staging/LICENSE");
+                contact, "License Text", "https://github.com/chpladmin/chpl-api/blob/staging/LICENSE");
```

Comment:
Should we/can we remove the terms of service link while we're in here? Or link to something that actually exists?

Reply:
We can! Do we have terms of service online or should I just remove the link?

Reply:
I don't think we have any TOS anywhere. Unless ONC has some I'd suggest we just remove it